Mathlib/Data/Polynomial/Degree/Lemmas.lean

```diff
@@ -315,6 +315,15 @@ theorem natDegree_map_eq_iff {f : R →+* S} {p : Polynomial R} :
   simp [h, h2, h3] -- simp doesn't rewrite in the hypothesis for some reason
   tauto
 
+theorem natDegree_pos_of_nz_nextCoeff (h : p.nextCoeff ≠ 0) : 0 < p.natDegree := by
```

Comment:
```suggestion
theorem natDegree_pos_of_nextCoeff_ne_zero (h : p.nextCoeff ≠ 0) : 0 < p.natDegree := by
```

Reply:
Fixed this (and similar names) in new commit